crypto/crypto/src/ed25519.rs

```diff
@@ -359,6 +359,7 @@ impl Signature for Ed25519Signature {
         self.0.to_bytes().to_vec()
     }
 
+    #[cfg(batch)]
```

Comment:
Did you intend this to be the following instead?
```suggestion
    #[cfg(feature = "batch")]
```

Reply:
:facepalm: